test/benchee_test.exs

```diff
@@ -225,6 +226,73 @@ defmodule BencheeTest do
     refute output =~ ~r/compar/i
   end
 
+  test "integration profiling can be deactivated" do
+    output =
+      capture_io(fn ->
+        Benchee.run(
+          %{"Sleeps" => fn -> :timer.sleep(10) end},
+          Keyword.merge(
+            @test_configuration,
+            profile_after: false
```

Comment:
I think we could leave out the config and just say it defaults to no profile, what do you think?

Reply:
Sure thing!